torch/onnx/utils.py

```diff
@@ -747,6 +742,14 @@ def _export(model, args, f, export_params=True, verbose=False, training=None,
                         opened_file.write(v)
             else:
                 raise RuntimeError("Unknown export type")
+
+            # Only run checker if we are using ONNX export type and
```

Comment:
* Remove this sentence. It just repeats what is obvious from reading the code immediately below.
* Add a comment explaining why the check on operator export type is here.